util/fibers/detail/fiber_interface.h

```diff
@@ -24,7 +24,7 @@ enum class Launch {
 namespace detail {
 
 using FI_ListHook =
-    boost::intrusive::slist_member_hook<boost::intrusive::link_mode<boost::intrusive::safe_link>>;
+    boost::intrusive::list_member_hook<boost::intrusive::link_mode<boost::intrusive::safe_link>>;
```

Comment:
👍🏻 I wonder how much it improves performance for heavy load with frequent disconnects 🙂